rotkehlchen/history/events/structures/base.py

```diff
@@ -253,7 +253,7 @@ def serialize_for_csv(self) -> dict[str, Any]:
             new_dict[key] = value
             if key == 'asset':  # input the amount/usd value right after the asset
                 new_dict['amount'] = balance['amount']
-                new_dict['usd_value'] = balance['usd_value']
+                new_dict[f'{currency.symbol_or_name().lower()}_value'] = balance['usd_value']
```

Comment:
don't do it since the frontend will need to handle specific cases and doesn't help. Call it `fiat_value`. Having variable keys is worst. Also this change needs an update in the api docs